orchagent/vnetorch.cpp

```diff
@@ -715,6 +734,73 @@ bool VNetBitmapObject::addIntf(const string& alias, const IpPrefix *prefix)
             SWSS_LOG_ERROR("Failed to create local VNET route entry, SAI rc: %d", status);
             throw std::runtime_error("VNet interface creation failed");
         }
+
+        intf.pfxMap.emplace(*prefix, intfPfxInfo);
+    }
+
+    return true;
+}
+
+bool VNetBitmapObject::removeIntf(const string& alias, const IpPrefix *prefix)
+{
+    SWSS_LOG_ENTER();
+
+    sai_status_t status;
+
+    if (intfMap_.find(alias) == intfMap_.end())
+    {
+        SWSS_LOG_ERROR("VNET '%s' interface '%s' doesn't exist", getVnetName().c_str(), alias.c_str());
+        return false;
+    }
+
+    auto& intf = intfMap_.at(alias);
+
+    if (prefix)
+    {
+        if (intf.pfxMap.find(*prefix) == intf.pfxMap.end())
+        {
+            SWSS_LOG_ERROR("VNET '%s' interface '%s' prefix '%s' doesn't exist",
+                    getVnetName().c_str(), alias.c_str(), prefix->getIp().to_string().c_str());
+            return false;
```

Comment:
Suggest to `return true` here. We don't want to retry in this case.

Reply:
Fixed.